app/workers/evss/disability_compensation_form/submit_form526.rb

```diff
@@ -30,11 +30,15 @@ class SubmitForm526
       # @param form_content [Hash] The form content that is to be submitted
       # @param uploads [Hash] The users ancillary uploads that will be submitted separately
       #
-      def perform(user_uuid, auth_headers, claim_id, form_content, uploads)
+
+      # rubocop:disable Metrics/MethodLength, Metrics/ParameterLists
+      def perform(user_uuid, auth_headers, claim_id, form_content, form4142, uploads)
         associate_transaction(auth_headers, claim_id, user_uuid) if transaction_class.find_transaction(jid).blank?
 
         response = service(auth_headers).submit_form(form_content)
 
+        submit_4142(form4142, response.claim_id, saved_claim.created_at) if form4142
```

Comment:
@egarreau : LOC "saved_claim.created_at" i guess must be "saved_claim(claim_id).created_at", else it will throw an error wrong number of arguments (given 0, expected 1)..

Reply:
thanks for the catch